http/src/request/channel/message/create_message.rs

```diff
@@ -214,6 +216,17 @@ impl<'a> CreateMessage<'a> {
         self
     }
 
+    /// Specify the ID of another message to create a reply to.
+    pub fn reply(mut self, other: MessageId) -> Self {
+        self.fields.message_reference.replace(MessageReference {
+            channel_id: Some(self.channel_id),
+            guild_id: None,
```

Comment:
It should be documented why the guild ID is not present for anyone who passes by this in the future